server/server.go

```diff
@@ -84,7 +90,7 @@ type Server struct {
 	cursors            *cursorManager
 	raftLogListenersMu sync.RWMutex
 	raftLogListeners   []RaftLogListener
-	authzEnforcer      *casbin.Enforcer
+	authzEnforcer      AuthzEnforcer
```

Comment:
I would probably make `AuthzEnforcer` a pointer here and then make its mutex just a value rather than pointer.

Reply:
fixed